crates/txtx-cli/src/cli/runbooks/mod.rs

```diff
@@ -819,28 +821,6 @@ pub async fn handle_run_command(
                 running_context.execution_context.execution_mode = RunbookExecutionMode::FullFailed;
             }
 
-            if let Some(RunbookState::File(state_file_location)) = runbook_state {
```

Comment:
I'm a bit confused by this removal - we're no longer saving state here?

Reply:
Fix in progress, I ran into a stashing snafu